entrypoint.sh

```diff
@@ -33,6 +33,11 @@ case $CLOUD in
     ;;
 esac
 
+if [ -f "/proc/1/cgroup" ]; then
```

Comment:
I'm not up to speed on what this if condition signifies, can you add some background detail?

Reply:
Sorry - I didn't mean to mix Dockerfile and entrypoint.sh into this PR.

I have it in my fork for these reasons:

1. Dockerfile is now a multistage build to make it simpler (no need for a one-line build and cleanup to keep the image small)
2. It will build drachtio based on the local files rather than pulling from origin repo

The entrypoint.sh script changes are required for a running container and cover conditions running in an AWS environment. The local IP you must bind to differs from the IP returned by the other calls to the AWS API.

The test for the file /proc/1/cgroup is to check if it is running inside a Docker instance. Although I have been reading further on this, I can see this method is not foolproof. Using an environment variable to use this form of check might be better.